src/PlusDataCollection/Andor/vtkPlusAndorCamera.h

```diff
@@ -0,0 +1,266 @@
+/*=Plus=header=begin======================================================
+  Program: Plus
+  Copyright (c) Laboratory for Percutaneous Surgery. All rights reserved.
+  See License.txt for details.
+=========================================================Plus=header=end*/
+
+#ifndef __vtkPlusAndorCamera_h
+#define __vtkPlusAndorCamera_h
+
+#include "vtkPlusDataCollectionExport.h"
+#include "vtkPlusDevice.h"
+
+/*!
+ \class vtkPlusAndorCamera
+ \brief Class for acquiring images from Andor cameras
+
+ Requires PLUS_USE_ANDOR_CAMERA option in CMake.
+ Requires the Andor SDK (SDK provided by Andor).
+
+ \ingroup PlusLibDataCollection.
+*/
+class vtkPlusDataCollectionExport vtkPlusAndorCamera: public vtkPlusDevice

```

Comment:
The naming convention for video devices is vtk<DeviceName>VideoSource. In this case it should be vtkPlusAndorVideoSource in so named .h/.cxx files